cmd/check_container.go

```diff
@@ -248,13 +248,15 @@ func init() {
 	checkContainerCmd.Flags().String("pyxis-api-token", "", "API token for Pyxis authentication (env: PFLT_PYXIS_API_TOKEN)")
 	viper.BindPFlag("pyxis_api_token", checkContainerCmd.Flags().Lookup("pyxis-api-token"))
 
-	checkContainerCmd.Flags().String("pyxis-host", "", fmt.Sprintf("Host to use for Pyxis submissions.\nThis will override Pyxis Env.\nOnly set this if you know what you are doing.\nIf you do set it, it should include just the host, and the URI path.\n(env: PFLT_PYXIS_HOST)"))
+	checkContainerCmd.Flags().String("pyxis-host", "", fmt.Sprintf("Host to use for Pyxis submissions.\nThis will override Pyxis Env.\n"+
```

Comment:
Four line breaks just seem like a lot. Totally a nit. Feel free to disregard.

--

As it pertains to adding an example value: 

If we're really concerned about partners using this when they shouldn't (as in, when they mean to use the other flag), then perhaps we need to massage how we indicate to the user that this is unlikely to be the flag they want. Our current verbiage (`Only set this if you know what you are doing.) doesn't really help in this regard. 

Not including an example value to prevent folks from using it when they shouldn't doesn't make sense.

If we really want to go that route, just mark the flag hidden. I'd prefer not to do this.

Reply:
I didn't change any of the verbiage here, or come up with it, it's existing. I just changed the formatting so the code was readable and not 800 characters of scroll city. For this flag I'd be fine with it being hidden, since it would be for us as developers to use to test potential issues between the internal/external instances or say a pyxis route change.

Reply:
I'll let @bcrochet chime in. I'll toss in the LGTM if Brad's good with it.